modules/ROOT/pages/_partials/include-install-tasks.adoc

```diff
@@ -1,12 +1,12 @@
 // tag::licenseKey[]
 == Insert the Mule License Key
 
-If you didn’t add the Mule license key during install, you can add it using the `rtfctl` command line utility.
+If you didn’t add the Mule license key during install, you can add it using the `rtfctl` command line utility or Helm.
 
-After the installation has completed succesfully, insert the Mule license key.
+Before you install the license key, Base64 encode it.
```

Comment:
<Channeling Kim> you should probably avoid verbing nouns...

Is there a way to say this differently? "Encode to base64 format"?

Reply:
Or hyphenate? "Base64-encode"

Reply:
oh good point